### PR TITLE
Add expired test

### PR DESCRIPTION
**Pull request by jonasfj**
As you pointed out we needed a test like this... Thanks! :)

### CODE REVIEW

File: test/remotevalidation_test.js [hunk 1]
```diff
@@ -191,4 +191,48 @@ suite("Remote Signature Validation", () => {
...
+      }
+    });
+    await myClient.resource();
+  });
+
+  test("auth with non-root user (expired)", async () => {
+    var clientId = slugid.v4();
+    var result = await helper.auth.createClient(clientId, {
+      scopes:       ['myapi:*'],
+      expires:      new Date(1998, 1, 1), // far out in the future
```
Comment (petemoore) [inline test/remotevalidation_test.js]: I think this should say `// far back in the past`

Comment (jhford) [inline test/remotevalidation_test.js]: yep!

**Comment by jhford** [comment]
r+ from me, i'll let you deploy when it makes sense to do so!

**Comment by djmitche** [comment]
@jonasfj does this still need to be landed?

**Comment by djmitche** [comment]
Hah, well, these tests now fail -- we appear not to ever check whether a client is expired.

**Comment by djmitche** [comment]
merged as part of #32

**Comment by djmitche** [comment]
(the merge commit got flattened in a rebase, oops)

**Comment by jonasfj** [comment]
awesome thanks for landing this....

**Pull request closed without merging**